components/filter/filter-overflow-group.js

```diff
@@ -105,9 +116,27 @@ class FilterOverflowGroup extends OverflowGroupMixin(RtlMixin(LitElement)) {
 				if (!filterSetValue) return;
 				filterSetValue.selected = change.selected;
 			});
+
+			this._getFilterParent(e.target.classList, filterSet)?.requestFilterChangeEvent(e.detail.allCleared, [dimension], { overflowEvent: true });
 		});
 	}
 
+	async _handleFilterLoadMore(e) {
+		const filterSet = this.querySelector(`d2l-filter-dimension-set[key=${e.detail.key}`);
+		if (!filterSet) return;
+		const filter = this._getFilterParent(e.target.classList, filterSet);
+		filter.requestFilterLoadMoreEvent(e.detail.key, e.detail.value, e.detail.loadMoreCompleteCallback);
+		this._handleSlotChange();
```

Comment:
Calling this when `load more` and `search` occurred did refresh the overflow group filter contents, but also some features were missing like having the selected items at the top of the list, and on search the selected items were all still visible. This can likely be simplified.